packages/next-app/src/app/metadata/layout.tsx

```diff
@@ -69,7 +69,16 @@ export default function Layout({ children }: React.PropsWithChildren) {
               </Sidebar>
             </div>
             <CheckSupportedChains
-              supportedChains={[Network.Goerli, Network.Polygon]}
+              supportedChains={[
+                // Network.Ethereum,
```

Comment:
need any help with this?

Reply:
nope, I just haven't deployed to Ethereum yet 😄